cookbooks/fb_postfix/attributes/default.rb

```diff
@@ -15,6 +15,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# RHEL 10 and later don't support hash maps anymore
+# https://src.fedoraproject.org/rpms/postfix/c/6a2621e4d73d59337dc64ba45922132286b841a1
+if node.centos_min_version?(10) || node.eln?
+  map_type = 'lmdb'
+else
+  map_type = 'hash'
+end
```

Comment:
shouldn't this just be el_min_version? this applies to RHEL too, right?

Reply:
Yeah, good call.